tests/test_rse.py

```diff
@@ -1280,15 +1278,9 @@ def test_swap_protocol(self, rucio_client):
         prots = rucio_client.get_protocols(protocol_rse)
         for p in prots:
             if p['scheme'] == 'MOCKA':
-                if p['domains']['lan']['read'] != 3:
-                    print('MOCKA with unexpected priority')
-                    print(prots)
-                    assert False
+                assert p['domains']['lan']['read'] == 3, f"MOCKA with unexpected priority: {p['domains']['lan']['read']}. Protocols: {prots}"
             if p['scheme'] == 'MOCKC':
-                if p['domains']['lan']['read'] != 1:
-                    print('MOCKC with unexpected priority')
-                    print(prots)
-                    assert False
+                assert p['domains']['lan']['read'] == 1, f"MOCKC with unexpected priority: {p['domains']['lan']['read']}. Protocols: {prots}"
```

Comment:
While logically equivalent, this change of assert cannot be considered purely cosmetic, and should not be in this commit - should be in 26c459c3915f444b658c84ed8a818912e226307a